### PR TITLE
New version: voidtools.Everything.Alpha version 1.5.0.1339a

### PR DESCRIPTION
**Pull request by krystofkrticka**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [X] Have you tested your manifest locally with `winget install --manifest <path>`? image
- [X] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/98541)

### CODE REVIEW

File: manifests/v/voidtools/Everything/Alpha/1.5.0.1339a/voidtools.Everything.Alpha.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Custom: /NORESTART
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://www.voidtools.com/Everything-1.5.0.1339a.x64-Setup.exe
+  InstallerSha256: 76A78E1F23BD4430AC78E7B70B7738B0867D38D6E414C7C2FF843E0B307A85BA
+  ProductCode: Everything 1.5a
```
Comment (mdanish-kh) [inline manifests/v/voidtools/Everything/Alpha/1.5.0.1339a/voidtools.Everything.Alpha.installer.yaml:22]:
```suggestion
  ProductCode: Everything 1.5a
  AppsAndFeaturesEntries:
    - DisplayName: Everything 1.5.0.1339a (x64)
      DisplayVersion: 1.5.0.1339
```
To match Control Panel / Apps&Features

File: manifests/v/voidtools/Everything/Alpha/1.5.0.1339a/voidtools.Everything.Alpha.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+- Architecture: x86
+  InstallerUrl: https://www.voidtools.com/Everything-1.5.0.1339a.x86-Setup.exe
+  InstallerSha256: FF04EF2EABA6227222098A9D43E57957A189ACBBED0600E3E11D68A83E549A1F
+  ProductCode: Everything 1.5a
```
Comment (mdanish-kh) [inline manifests/v/voidtools/Everything/Alpha/1.5.0.1339a/voidtools.Everything.Alpha.installer.yaml:26]:
```suggestion
  ProductCode: Everything 1.5a
  AppsAndFeaturesEntries:
    - DisplayName: Everything 1.5.0.1339a (x86)
      DisplayVersion: 1.5.0.1339
```

File: manifests/v/voidtools/Everything/Alpha/1.5.0.1339a/voidtools.Everything.Alpha.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+Publisher: voidtools
+PublisherUrl: https://www.voidtools.com
+PublisherSupportUrl: https://www.voidtools.com/support/everything/
+PrivacyUrl: https://www.voidtools.com/privacy
+Author: voidtools
+PackageName: Everything (Alpha)
+PackageUrl: https://www.voidtools.com/forum/viewtopic.php?f=12&t=9787
+License: MIT License
+LicenseUrl: https://www.voidtools.com/License.txt
+Copyright: Copyright (C) 2021 voidtools
```
Comment (mdanish-kh) [inline manifests/v/voidtools/Everything/Alpha/1.5.0.1339a/voidtools.Everything.Alpha.locale.en-US.yaml:16]:
```suggestion
Copyright: Copyright (C) 2022 voidtools
```
According to https://www.voidtools.com/License.txt

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

**Review by mdanish-kh — DISMISSED** [review]

**Comment by krystofkrticka** [comment]
@mdanish-kh Shouldn't the copyright tag state: Copyright (C) 2023 voidtools instead of Copyright (C) 2022 voidtools or Copyright (C) 2021 voidtools because when I open about dialog of everything it states copyright 2023

File: manifests/v/voidtools/Everything/Alpha/1.5.0.1339a/voidtools.Everything.Alpha.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+Publisher: voidtools
+PublisherUrl: https://www.voidtools.com
+PublisherSupportUrl: https://www.voidtools.com/support/everything/
+PrivacyUrl: https://www.voidtools.com/privacy
+Author: voidtools
+PackageName: Everything (Alpha)
+PackageUrl: https://www.voidtools.com/forum/viewtopic.php?f=12&t=9787
+License: MIT License
+LicenseUrl: https://www.voidtools.com/License.txt
+Copyright: Copyright (C) 2022 voidtools
```
Comment (krystofkrticka) [inline manifests/v/voidtools/Everything/Alpha/1.5.0.1339a/voidtools.Everything.Alpha.locale.en-US.yaml:16]:
```suggestion
Copyright: Copyright (C) 2023 voidtools
```

**Comment by mdanish-kh** [comment]
> @mdanish-kh Shouldn't the copyright tag state: Copyright (C) 2023 voidtools instead of Copyright (C) 2022 voidtools or Copyright (C) 2021 voidtools because when I open about dialog of everything it states copyright 2023

I got that year from the CopyrightUrl used in the manifest (https://www.voidtools.com/License.txt). As the publisher didn't update it on the official website, I didn't want to bump it in the manifest. But I guess the publisher forgot to update it on the website as the downloaded file shows the correct year:

```
 ~ (Get-Item .\Everything-1.5.0.1339a.x64-Setup.exe).VersionInfo.LegalCopyright
Copyright © 2023 voidtools
```

**Review by mdanish-kh — APPROVED** [review]

**Comment by krystofkrticka** [comment]
It looks like it. Maybe it's also caused by the fact that there wasn't update for stable version of everything and that's probably why developer didn't bother with updating copyright in license page.

**Pull request merged**